File: cloud/dynamic-site/gt-cs-1331-py/app/gtcs1331/solutions/solution109.py
'''Solution 1.9 implementation'''
from gtcs1331.solution_api.solution import Solution

class Solution109(Solution):
    '''Solution 1.9'''
    id = "1.9"
    title = "Rectangle measure"

    def __answer(self):
        width = 4.5
        length = 7.9
        perimeter = 2 * (width + length)
        area = width * length

        return [
            f"perimeter = {perimeter:.6f}".rstrip('0').rstrip('.'),
            f"area = {area:.6f}".rstrip('0').rstrip('.')]

    answer = property(__answer)

File: cloud/dynamic-site/gt-cs-1331-csharp/solutions/Solution109.cs
using _4MeOnWeb.SolutionAPI;

namespace _4MeOnWeb.Solutions {
  public class Solution109 : ImmutableSolution {
    public Solution109() :
        base("1.9", "Rectangle measure") {
    }

    public override IEnumerable<string> Answers {
      get {
        double width = 4.5;
        double length = 7.9;
        double perimeter = 2 * (width + length);
        double area = width * length;

        return new string[] {
          $"perimeter = {perimeter:f6}".TrimEnd(new[]{'0','.' }),
          $"area = {area:f6}".TrimEnd(new[]{'0','.' })
        };
      }
    }
  }
}